mscore/plugin/api/playevent.h

```diff
@@ -0,0 +1,118 @@
+//=============================================================================
+//  MuseScore
+//  Music Composition & Notation
+//
+//  Copyright (C) 2019 Werner Schweer and others
+//
+//  This program is free software; you can redistribute it and/or modify
+//  it under the terms of the GNU General Public License version 2
+//  as published by the Free Software Foundation and appearing in
+//  the file LICENCE.GPL
+//=============================================================================
+
+#ifndef __PLUGIN_API_PLAYEVENT_H__
+#define __PLUGIN_API_PLAYEVENT_H__
+
+#include "libmscore/noteevent.h"
+#include "elements.h"
+
+namespace Ms {
+namespace PluginAPI {
+
+class Note;
+
+//---------------------------------------------------------
+//   PlayEvent
+//    Wrapper class for internal Ms::NoteEvent
+//
+//   This is based on the wrapper in excerpt.h.
```

Comment:
Could you please add `\since MuseScore 3.3` statement here too (for the whole `PlayEvent` class)?

Reply:
fixed.